### PR TITLE
Use current filename for title.

### PR DESCRIPTION
**Pull request by ayudemura**

### CODE REVIEW

**Comment by summasmiff** [comment]
+1 :cake:

**Comment by jbonewits** [comment]
+1

File: LexLadiesGistCreator.py [hunk 1]
```diff
@@ -12,8 +12,9 @@ def run(self, edit, cmd=None):
         gist_content = self.view.substr(region)
         line = self.view.line(region)
         url = 'https://api.github.com/gists'
+        filename = self.view.file_name().split('/')[-1]
         files = {
-          'file1.txt': {
+            filename: {
```
Comment (cheshire137) [inline LexLadiesGistCreator.py:17]: Dat indentation, tho.

**Comment by cheshire137** [comment]
IN WE GO!

**Pull request merged**